README.md

```diff
@@ -133,7 +140,7 @@ To enable debugging for the underlying library, pass {penpalDebug: true} as the
 PublisherInterface.build(iframe, {penpalDebug: true});
 ```
 
-This will display messages in the console about Penpal and the communication between the iframe and the main page.
+This will display messages in the console from Penpal and the communication between the iframe and the main page.
```

Comment:
Discussed changing "from Penpal and the communication" to "from Penpal about the communication"